packages/docs/content-static/examples/rag-pipeline.md

```
@@ -0,0 +1,130 @@
+| `llm_1` | `llm_openai` | Receives the question + retrieved context and generates an answer using GPT-4o. |
+| `target_1` | `response` | Returns the answer to the caller. |
+
+## Start the pipeline
+
+```bash
+rocketride start --pipeline ./rag.pipe
+```
+
+The engine prints the webhook URL and public auth key:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)